root/lib/systemd/system/titus-metatron-sync@.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Metatron sync for container %i
+ConditionPathIsDirectory=/var/lib/titus-inits/%i/ns
+
+[Service]
+Environment=TITUS_PID_1_DIR=/var/lib/titus-inits/%i
+EnvironmentFile=/var/lib/titus-environments/%i.env
+ExecStart=/apps/titus-executor/bin/titus-nsenter /titus/metatron/bin/titusRefresh.sh
```

Comment:
Why not `docker-runc --root /var/run/docker/runtime-runc/moby exec 203ebda90e5e848582e65a06259defa10b53984f43f2a1392838f7b190d3e7bf touch /foo`

That way, it's isolate just like the container, and you don't need to do PrivateTmp? 

You can grab 203ebda90e5e848582e65a06259defa10b53984f43f2a1392838f7b190d3e7bf from EnvironmentFile?